### PR TITLE
Revamp ember app

### PR DESCRIPTION
**Pull request by Tav0**
Creates a conference.
Router structure follows Ember guidelines.
Minor tweaks on Conference model and serializer.
Clean up of unneeded files.
Removing extra code.

### CODE REVIEW

**Comment by Tav0** [comment]
Removed the [WIP] tag due to being mostly complete. Some of the code is still commented and the conference/index folder is useless, but that all this minor changes will be done in the next ticket I'm working on.

**Comment by Ghalko** [comment]
Can we still not post conferences?
screen shot 2016-07-07 at 8 43 04 am

Okay, really that was because errors are not being forwarded, or used, in the Ember app from Django.

**Comment by Ghalko** [comment]
There are still mention of nodes in the code, are they only in reference to the OSF?

File: app/models/conference.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
-  siteUrl: attr('string'),
-  logoUrl: attr('string'),
+export default Model.extend({
+    title: attr('string'),
+    city: attr('string'),
+    state: attr('string'),
+    country: attr('string'),
+    start: attr('isodate', { defaultValue : (new Date()).toISOString() }),
+    end: attr('isodate', { defaultValue : (new Date()).toISOString() }),
+    submissionstart: attr('isodate', { defaultValue : (new Date()).toISOString() }),
```
Comment (Ghalko) [inline app/models/conference.js:12]: Please consider using camel case for variables in JS. Why did you change to all lower case?

**Comment by Ghalko** [comment]
@Tav0 Please update the description to describe what is done, then it will be ready to merge.

**Comment by Tav0** [comment]
@Ghalko The node mentions are all about submission which is not related to this PR, since this is only for Conference related stuff.
I updated the description.

File: app/models/conference.js (hunk 1 above)
Comment (Tav0) [inline app/models/conference.js:12]: Will be changing it back to camel case since now we have a PR for the back-end fix.
